packages/rum/src/domain/getReplayLink.ts

```diff
@@ -0,0 +1,20 @@
+import type { RumConfiguration, RumSessionManager, ViewContexts } from '@datadog/browser-rum-core'
+
+export function getReplayLink(
+  configuration: RumConfiguration,
+  sessionManager: RumSessionManager,
+  viewContexts: ViewContexts,
+  subdomain: string
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** Not all site require a subdomain, so this argument should at least be optional

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think letting the customer specify a subdomain complicates the API a bit. Could we avoid that and use sensible defaults? Ex:

| site |  subdomain |
|---|---|
| datadoghq.com | `app` |
| datad0g.com | `dd` |
| anything else | no subdomain |

Reply:
To clarify: I'm not sure if we should let the customer set the subdomain at all. I think we could start without it, and if there is some need in the future, adjust the API.